templates/kubernetes/config.yml

```diff
@@ -0,0 +1,5 @@
+name: Kubernetes
+description: |
+  Kubernetes service
+version: v0.1.0-rancher1
+category: Rancher System Services
```

Comment:
Won't it be nice to have rancher environment as a category too. Under
applications,  we will STACKS, SYSTEM, ENVIROMEMENT.  environment will have
k8s, mesos, cattle and swarm. So all environments can be easily upgraded.
On May 25, 2016 6:28 PM, "Alena Prokharchyk" <notifications@github.com>
wrote:

> In templates/kubernetes/config.yml
> <https://github.com/rancher/rancher-catalog/pull/150#discussion_r64664693>
> :
>
> > @@ -0,0 +1,5 @@
> > +name: Kubernetes
> > +description: |
> > +  Kubernetes service
> > +version: v0.1.0-rancher1
> > +category: Rancher System Services
>
> @vincent99 <https://github.com/vincent99> introduced "Rancher System
> Services" category so we can filter out k8s system template from the
> Catalog UI. Other naming suggestions are welcome.
>
> —
> You are receiving this because you are subscribed to this thread.
> Reply to this email directly or view it on GitHub
> <https://github.com/rancher/rancher-catalog/pull/150/files/dd1b65bdcffee545eeef9ad43c869c8a63a7263f#r64664693>
>